Lib/ufo2ft/featureWriter/__init__.py

```diff
@@ -0,0 +1,35 @@
+from __future__ import (
+    print_function, division, absolute_import, unicode_literals)
+
+
+def liststr(glyphs):
+    """Return string representation of a list of glyph names."""
+    return "[%s]" % " ".join(glyphs)
+
+
+class BaseFeatureWriter(object):
+    """Abstract features writer"""
+
+    def __init__(self, font, features=(), scriptLangs=()):
+        self.font = font
+        self.features = features
+        self._scriptLangs = scriptLangs
+
+    def write(self, **kwargs):
+        """Write features and class definitions"""
+        raise NotImplementedError
+
+    def _get_scriptLangs(self):
+        if self._scriptLangs:
+            return self._scriptLangs
+        else:
+            return {"DFLT": ("dflt", )}
+
+    scriptLangs = property(_get_scriptLangs)
```

Comment:
what's the advantage of having `scriptLangs` as a property getter rather than a regular instance attribute? to make it read-only? or allow subclasses to return a different default value?

Reply:
To return a default value, I guess

Reply:
ok. A class attribute, optionally overridden on the instance via constructor argument, would achieve the same. But this works too.